Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.21
+FROM alpine:3.22
```

Comment:
Lets keep this at 3.21, 3.22 just barely got release so there could still be bugs.

Will bump for next Helm release

Reply:
I think the risks are very small, but I've adjusted it.